run_evaluation.py

```diff
@@ -3,49 +3,53 @@
 import imp
 imp.reload(hourglass)
 from hourglass import HourglassNet
-import data_generator
-imp.reload(data_generator)
-from data_generator import DataGenerator
-import evaluation
-imp.reload(evaluation)
-from evaluation import Evaluation
 from constants import *
 import matplotlib.pyplot as plt
+import os
```

Comment:
Just making a few suggestions to add a print for start and end time 
```suggestion
import os
import time
```